src/Editor/Image/Components/Cropper.tsx

```diff
@@ -2,26 +2,19 @@ import React, { FC, useMemo, useRef, useState } from 'react';
 import { CropperRef, Cropper } from 'react-advanced-cropper';
 import { WhpptImageCrop } from '../Model/ImageData';
 
-export const defaultCoordinates = {
-  height: 150,
-  width: 250,
-  top: 20,
-  left: 10,
-};
-
 type WhpptCropperProps = {
   value: WhpptImageCrop;
   onChange: (data: WhpptImageCrop) => void;
 };
 
 export const getLandscapeRatio = ratio => {
   const { w, h } = ratio;
-  return w / h;
+  return w >= h ? w / h : h / w;
```

Comment:
I don't think this is correct. We should not switch orientation based on size but rather user input

Reply:
If the developer or user chooses the ratio that does not align to their values then its up to them to fix when id doesn't behave as expected.